### ISSUE TITLE
[Cisco_duo] feature request - additional enrichment for access_device

### ISSUE THREAD

**Issue by eriroley** [elastic/integrations#11333]
using API v1
information in the "auth_device" is enriched to include "as.number" and "as.organization"
![Image](https://github.com/user-attachments/assets/1bfcd65a-6244-422d-8bce-b75bf18ae178)

but devices/ip addresses in the "access_device" are not enriched to the same level
![Image](https://github.com/user-attachments/assets/0a08967e-2074-459e-bbdd-4056f4559478)

based on event.original - it looks like the location data is parsed directly from the original string, and no enrichment on the access_device.ip is done at all

**Comment by chemamartinez** [comment]
Hey @eriroley,

Taking a look at the ingest pipeline for these events, I can see the following sequence:

- IP address from the `access_device` field is mapped as the ECS `source.ip` field: https://github.com/elastic/integrations/blob/main/packages/cisco_duo/data_stream/auth/elasticsearch/ingest_pipeline/default.yml#L96-L99
- ASN enrichment is done a few lines below, adding them to the `source.as` field: https://github.com/elastic/integrations/blob/main/packages/cisco_duo/data_stream/auth/elasticsearch/ingest_pipeline/default.yml#L166-L185

In other words, the enrichment is done the same as for the `auth_device` IPs, but in the case of `access_device`, they are mapped as the `source` ECS field.

Does it make sense to you?
